llvm/lib/Analysis/CFG.cpp

```diff
@@ -204,6 +221,43 @@ bool llvm::isPotentiallyReachableFromMany(
   return false;
 }
 
+template <class T> class SingleEntrySet {
+public:
+  using iterator = T *;
+  using const_iterator = const T *;
+
+  SingleEntrySet(T Elem) : Elem(Elem) {}
+
+  bool contains(T Other) const { return Elem == Other; }
+
+  iterator begin() { return &Elem; }
+  iterator end() { return &Elem + 1; }
```

Comment:
I don't think the non-const methods are necessary.

Reply:
Done.